src/app/shared/bioschema.service.ts

```diff
@@ -80,9 +81,6 @@ export class BioschemaService {
         '@type': 'Organization',
         name: tool.namespace,
       };
-      if (tool.email) {
```

Comment:
I removed this and the corresponding portion for workflows below because an entry's email is the email of the author